code/game/objects/items/deployable/bot.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/deployable/bot
+	name = "deployable bot" //should never be visible
+	desc = "if you're seeing this contact coders ASAP"
+	dense_deployment = TRUE
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
These should be large so that they can't be carried around in backpacks imo

Reply:
Bots are bulky now 👍